src/ResourceManager/Network/documentation/current-breaking-changes.md

```diff
@@ -38,4 +38,19 @@
     https://github.com/Azure/azure-powershell/blob/dev/documentation/breaking-changes/breaking-change-template.md
 -->
 
-## Current Breaking Changes
```

Comment:
Please revert all changes here - we will automatically move upcoming breaking changes to current breaking changes when we release.

Reply:
Okay, good to know.